cmd/kubelet/kubelet.go

```diff
@@ -36,11 +36,6 @@ import (
 
 func main() {
 	command := app.NewKubeletCommand()
-
-	// kubelet uses a config file and does its own special
-	// parsing of flags and that config file. It initializes
-	// logging after it is done with that. Therefore it does
-	// not use cli.Run like other, simpler commands.
```

Comment:
Can you also eliminate the run function? It serves no purpose anymore.

Reply:
And remove `command.SetGlobalNormalizationFunc(cliflag.WordSepNormalizeFunc)`?

That is done by cli.Run: https://github.com/kubernetes/kubernetes/blob/fa16bf8e121e9a9ce0a8b92d96a39c986152c484/staging/src/k8s.io/component-base/cli/run.go#L92

Reply:
I have updated. could you please take a look again, thanks? @pohly